### ISSUE TITLE
Which versions modules are used for this project?

### ISSUE THREAD

**Issue by lllkiller-jlll** [cogsys-tuebingen/spindoe#2]
Hi. 

I'm trying to use this project.

I set python 3.9 environment on conda and I installed modules default pip command.

But the result image is different from README. My result is below.

![Image](https://github.com/user-attachments/assets/58734fb7-9d7b-497c-a0a9-6c0454a96ad9)

And spin result of debug function is "None".

Unexpected result is due to version difference? or is there any mistake my steps?

please let me know.

Thanks.

**Comment by lllkiller-jlll** [comment]
I attach my package list.


Package                  Version
------------------------ -----------
aiohappyeyeballs         2.6.1
aiohttp                  3.11.16
aiosignal                1.3.2
annotated-types          0.7.0
async-timeout            5.0.1
attrs                    25.3.0
certifi                  2025.1.31
charset-normalizer       3.4.1
click                    8.1.8
contourpy                1.3.0
cycler                   0.12.1
docker-pycreds           0.4.0
eval_type_backport       0.2.2
filelock                 3.18.0
fonttools                4.57.0
frozenlist               1.5.0
fsspec                   2025.3.2
gitdb                    4.0.12
GitPython                3.1.44
idna                     3.10
importlib_resources      6.5.2
Jinja2                   3.1.6
joblib                   1.4.2
kiwisolver               1.4.7
lightning-utilities      0.14.3
MarkupSafe               3.0.2
matplotlib               3.9.4
mpmath                   1.3.0
multidict                6.3.2
networkx                 3.2.1
numpy                    2.0.2
nvidia-cublas-cu12       12.4.5.8
nvidia-cuda-cupti-cu12   12.4.127
nvidia-cuda-nvrtc-cu12   12.4.127
nvidia-cuda-runtime-cu12 12.4.127
nvidia-cudnn-cu12        9.1.0.70
nvidia-cufft-cu12        11.2.1.3
nvidia-curand-cu12       10.3.5.147
nvidia-cusolver-cu12     11.6.1.9
nvidia-cusparse-cu12     12.3.1.170
nvidia-cusparselt-cu12   0.6.2
nvidia-nccl-cu12         2.21.5
nvidia-nvjitlink-cu12    12.4.127
nvidia-nvtx-cu12         12.4.127
opencv-python            4.11.0.86
packaging                24.2
pandas                   2.2.3
pillow                   11.1.0
pip                      25.0
platformdirs             4.3.7
propcache                0.3.1
protobuf                 5.29.4
psutil                   7.0.0
pydantic                 2.11.2
pydantic_core            2.33.1
pyparsing                3.2.3
python-dateutil          2.9.0.post0
pytorch-lightning        1.9.3
pytz                     2025.2
PyYAML                   6.0.2
requests                 2.32.3
scikit-learn             1.6.1
scipy                    1.13.1
sentry-sdk               2.25.1
setproctitle             1.3.5
setuptools               75.8.0
six                      1.17.0
smmap                    5.0.2
sympy                    1.13.1
threadpoolctl            3.6.0
torch                    2.6.0
torchmetrics             1.7.1
torchvision              0.21.0
tqdm                     4.67.1
triton                   3.2.0
typing_extensions        4.13.1
typing-inspection        0.4.0
tzdata                   2025.2
urllib3                  2.3.0
wandb                    0.19.9
wheel                    0.45.1
yarl                     1.19.0
zipp                     3.21.0

**Comment by yongyong25** [comment]
Hello. I'm a student. and I'm trying to use this project, too.
I also had the same problem. I use a Windows environment, but the environment you provided seems to be Linux. 
Could there be a problem due to the difference?

**Comment by 9527ys** [comment]
Hello, I have also encountered a similar problem. Could you please tell me if you have solved this issue?
